src/components/NodeLink/NodeLink.vue

```diff
@@ -120,10 +129,22 @@ export default {
         isDirected,
         isMultiEdge,
         attributes,
+        linkWidthVariable,
+        linkColorVariable,
         labelVariable,
         colorVariable,
       };
-    }
+    },
+    linkWidthScale() {
+      return d3.scaleLinear()
+        .domain(
+          [
+            d3.min(this.graphStructure.links.map(d => d[this.linkWidthVariable])),
+            d3.max(this.graphStructure.links.map(d => d[this.linkWidthVariable]))
```

Comment:
It would be good to lift the two dependencies out into a destructuring declaration at the top of the function, just to make it visually more apparent what these dependencies are, but that's a minor style improvement that's not blocking this PR.

But hopefully you've gotten a taste for why computed props are nice to work with :cupid: